modules/OFConnectionManager2/OFConnectionManager.mk

```diff
@@ -0,0 +1,14 @@
+
+###############################################################################
+#
+# Inclusive Makefile for the OFConnectionManager module.
+#
+# Autogenerated 2015-01-20 16:47:17.185057
+#
+###############################################################################
+OFConnectionManager_BASEDIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+include $(OFConnectionManager_BASEDIR)/module/make.mk
+include $(OFConnectionManager_BASEDIR)/module/auto/make.mk
+include $(OFConnectionManager_BASEDIR)/module/src/make.mk
+include $(OFConnectionManager_BASEDIR)/utest/_make.mk
+
```

Comment:
The original plan was to 'cp -r OFConnectionManager OFConnectionManager2', make changes, and then 'mv OFConnectionManager2 OFConnectionManager' when we were going to move the code in.  I'll check with Jeff to see how we are ultimately going to bring this code in.